test/data/testinit.js

```diff
@@ -346,16 +346,16 @@ this.includesModule = function( moduleName ) {
 
 	var excludedModulesPart, excludedModules;
 
-	// A short-cut for the slim build, e.g. "4.0.0-pre slim"
-	if ( jQuery.fn.jquery.indexOf( " slim" ) > -1 ) {
+	// A short-cut for the slim build, e.g. "4.0.0-pre+slim"
+	if ( jQuery.fn.jquery.indexOf( "+slim" ) > -1 ) {
 
 		// The module is included if it does NOT exist on the list
 		// of modules excluded in the slim build
-		return slimBuildFlags.indexOf( "-" + moduleName ) === -1;
+		return excludedFromSlim.indexOf( moduleName ) === -1;
 	}
 
-	// example version for `grunt custom:-deprecated`:
-	// "4.0.0-pre -deprecated,-deprecated/ajax-event-alias,-deprecated/event"
+	// example version for `npm run build -- -e deprecated`:
+	// "v4.0.0-pre+14dc9347 -deprecated,-deprecated/ajax-event-alias,-deprecated/event"
```

Comment:
Let's not change the format on `3.x-stable`. This will most likely also require some changes in `build.js` and possibly in a README or two.

Reply:
I'd still like the SHA, but that only affects development , but I won't change the version for releases (i.e. the slim identifier).

Reply:
That sounds sensible.

Maybe you can also update the example to use something like `v3.7.2-pre+...` since the formats between 3.x & 4.x slightly differ and thus it's a bit confusing to see `v4` here in the example.